### PR TITLE
Updated response-body of /v1/regard-application

### PR DESCRIPTION
**Pull request by sunil-kumar-meena**
Fixes
https://github.com/openBackhaul/ApplicationLayerTopology/issues/570

reviewer @PrathibaJee

### CODE REVIEW

File: server/service/IntegerProfileService.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+
+/**
+ * Returns the configured value of the Integer
+ *
+ * uuid String 
+ * returns inline_response_200_32
+ **/
+exports.getIntegerProfileIntegerValue = async function(url) {
+  const value = await fileOperation.readFromDatabaseAsync(url);
```
Comment (PrathibaJee) [inline server/service/IntegerProfileService.js:25]: fileOperation is not defined. please import it from the JSONDriver.

File: server/service/IntegerProfileService.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+
+/**
+ * Returns the minimum value of the Integer
+ *
+ * uuid String 
+ * returns inline_response_200_30
+ **/
+exports.getIntegerProfileMinimum = async function(url) {
+  const value = await fileOperation.readFromDatabaseAsync(url);
```
Comment (PrathibaJee) [inline server/service/IntegerProfileService.js:53]: fileOperation is not defined. please import it from the JSONDriver.

File: server/service/IntegerProfileService.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+/**
+ * Returns the purpose of the Integer
+ *
+ * uuid String 
+ * returns inline_response_200_28
+ **/
+exports.getIntegerProfilePurpose = function(url) {
+  return new Promise(async function (resolve, reject) {
+    try {
+      var value = await fileOperation.readFromDatabaseAsync(url);
```
Comment (PrathibaJee) [inline server/service/IntegerProfileService.js:69]: fileOperation is not defined. please import it from the JSONDriver.

File: server/service/IntegerProfileService.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+ * uuid String 
+ * no response value expected for this operation
+ **/
+exports.putIntegerProfileIntegerValue = async function (body, url, uuid) {
+  let profile = await IntegerProfile.getIntegerProfile(uuid);
+  let maximumIntegerValue = profile.integerProfilePac.integerProfileCapability.maximum;
+  let minimumIntegerValue = profile.integerProfilePac.integerProfileCapability.minimum;
+  let value = body["integer-profile-1-0:integer-value"];
+  if (value >= maximumIntegerValue || value <= minimumIntegerValue){
+    throw new createHttpError.BadRequest(`integer-profile-1-0:integer-value must be in range between ${minimumIntegerValue} and ${maximumIntegerValue}`)
```
Comment (PrathibaJee) [inline server/service/IntegerProfileService.js:113]: 'createHttpError' is not defined

File: server/service/IntegerProfileService.js [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+/**
+ * Configures value of the Integer
+ *
+ * body Integerprofileconfiguration_integervalue_body 
+ * uuid String 
+ * no response value expected for this operation
+ **/
+exports.putIntegerProfileIntegerValue = async function (body, url, uuid) {
+  let profile = await IntegerProfile.getIntegerProfile(uuid);
```
Comment (PrathibaJee) [inline server/service/IntegerProfileService.js:108]: 'IntegerProfile' is not defined. Import Integer profile from the application pattern package.

File: server/service/IntegerProfileService.js [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+
+/**
+ * Returns the unit of the Integer
+ *
+ * uuid String 
+ * returns inline_response_200_29
+ **/
+exports.getIntegerProfileUnit = async function(url) {
+  const value = await fileOperation.readFromDatabaseAsync(url);
```
Comment (PrathibaJee) [inline server/service/IntegerProfileService.js:93]: fileOperation is not defined. please import it from the JSONDriver.

File: server/service/IntegerProfileService.js [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+ **/
+exports.putIntegerProfileIntegerValue = async function (body, url, uuid) {
+  let profile = await IntegerProfile.getIntegerProfile(uuid);
+  let maximumIntegerValue = profile.integerProfilePac.integerProfileCapability.maximum;
+  let minimumIntegerValue = profile.integerProfilePac.integerProfileCapability.minimum;
+  let value = body["integer-profile-1-0:integer-value"];
+  if (value >= maximumIntegerValue || value <= minimumIntegerValue){
+    throw new createHttpError.BadRequest(`integer-profile-1-0:integer-value must be in range between ${minimumIntegerValue} and ${maximumIntegerValue}`)
+  }
+  await fileOperation.writeToDatabaseAsync(url, body, false);
```
Comment (PrathibaJee) [inline server/service/IntegerProfileService.js:115]: fileOperation is not defined. please import it from the JSONDriver.

**Review by PrathibaJee — CHANGES_REQUESTED** [review]
Please fix the review comments. Also after fixing the review comments, perform unit testing for the integer profile.

File: server/service/IntegerProfileService.js [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+
+/**
+ * Returns the name of the Integer
+ *
+ * uuid String 
+ * returns inline_response_200_27
+ **/
+exports.getIntegerProfileIntegerName = async function(url) {
+  const value = await fileOperation.readFromDatabaseAsync(url);
```
[the remark this answers is not in the document]
Comment (sunil-kumar-meena) [reply server/service/IntegerProfileService.js:14]: this is fixed

**Review by PrathibaJee — APPROVED** [review]
Thanks for fixing the review comments.

**Pull request merged**